package/MDAnalysis/lib/_augment.pyx

```diff
@@ -0,0 +1,353 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+#
+
+import cython
+import numpy as np
+from .mdamath import triclinic_vectors
+
+from libcpp.vector cimport vector
+from libc.math cimport sqrt
+
+__all__ = ['augment_coordinates', 'undo_augment']
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def augment_coordinates(float[:, ::1] coordinates, float[:] box, float r):
+    """Calculates the relevant images of particles which are within a
+    distance 'r' from the box walls
+
+    Only the relevant images are generated i.e. for every set of
+    coordinates close to the box boundary,
+    corresponding images which are close to the opposite face and
+    outside the central cell are generated. If the particle
+    is close to more than one box walls,
+    images along the diagonals are also generated ::
+
+

```

Comment:
I think this needs to be indented by one space at least so that it is correctly rendered as preformatted text.